tools/sync-data-plane-api-sha

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import subprocess
+import time
+
+import github3
+
+
+def get_last_commit_sha(repo):
+    for commit in repo.iter_commits():
+        return commit.sha
+
+
+def get_sign_off_line():
+    name = subprocess.check_output(['git', 'config', 'user.name']).decode('utf-8').strip()
+    email = subprocess.check_output(['git', 'config', 'user.email']).decode('utf-8').strip()
+    return f'Signed-off-by: {name} <{email}>'
+
+
+bazel_path = 'bazel/repository_locations.bzl'
+
+parser = argparse.ArgumentParser(description='sync the latest data-plane-api commit SHA to envoy')
+parser.add_argument('--user')
+parser.add_argument('--password')
+parser.add_argument('--branch', help='branch on fork to push changes to', default='data-plane-api-sha-updates')
+args = parser.parse_args()
+
+gh = github3.login(args.user, args.password)
```

Comment:
this won't work for those of us with 2FA enabled, right? i would suggest using github's token feature or just calling out to system git.